code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -1324,6 +1324,8 @@
 		return FALSE
 	var/mob/living/carbon/xenomorph/X = owner
 	var/mob/living/carbon/victim = A //target of ability
+	if(X.status_flags & INCORPOREAL)
+		return FALSE
```

Comment:
Early return as soon as practical. In this case we don't care about typecasting the owner or ther victim, or even parent honestly, so you can check this before anything else.